tslint.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Maybe update this to match https://github.com/secondstreet/template-builder/blob/master/tslint.json? (If you add the prettier rule and plugin, update the lint-staged config to match as well.)

Reply:
I agree it would be a good idea to unify but given that we have multiple tslint configs across multiple projects, I'd like to keep this PR concerned with what it currently is, and handle tslint config unification across everywhere as its own project.